README.md

```diff
@@ -26,37 +32,37 @@ There are two supported ways of developing apps using AeroGear for Android: Andr
 ### Android Studio
 
 Add to your application's `build.gradle` file
-```
+
+```groovy
 dependencies {
-  compile 'org.jboss.aerogear:aerogear-android-core:3.0.0'
-  //My other dependencies
+  compile 'org.jboss.aerogear:aerogear-android-core:3.1.0'
 }
 ```
 
 ### Maven
 
 Include the following dependencies in your project's `pom.xml`
 
-```
+```xml
 <dependency>
   <groupId>org.jboss.aerogear</groupId>
   <artifactId>aerogear-android-core</artifactId>
-  <version>3.0.0</version>
+  <version>3.1.0</version>
   <scope>provided</scope>
   <type>jar</type>
 </dependency>
 
 <dependency>
   <groupId>org.jboss.aerogear</groupId>
   <artifactId>aerogear-android-core</artifactId>
-  <version>3.0.0</version>
+  <version>3.1.0</version>
   <type>aar</type>
 </dependency>
 ```
 
 ## Documentation
 
-For more details about the current release, please consult [our documentation](http://aerogear.org/docs/guides/aerogear-android/).
+For more details about the current release, please consult [our documentation](https://aerogear.org/android/).
```

Comment:
You changed the link here for a new one, but there is another `http://aerogear.org/docs/guides/aerogear-android/` in the table at the beginning. Should that be changed with this new URL too?